### PR TITLE
Support for notifications.

### PR DESCRIPTION
**Pull request by joshuaedwardcrowe**
Added property for Notification API.

### CODE REVIEW

File: src/System.js [hunk 1]
```diff
@@ -140,7 +140,9 @@ var System = {
 
 		webgl: ( function () { try { return !! window.WebGLRenderingContext && !! document.createElement( 'canvas' ).getContext( 'experimental-webgl' ); } catch( e ) { return false; } } )(),
 
-		worker: !! window.Worker
+		worker: !! window.Worker,
+		
+		notifications: !! window.Notification && 'Notification' in Window
```
Comment (mrdoob) [inline src/System.js]: Shouldn't it just be...

```js
notifications: 'Notification' in Window
```

Comment (Crowes) [inline src/System.js]: I would assume both property accessors are identical in function, but I would also go for the latter.

Comment (mrdoob) [inline src/System.js]: Cool! Could you update the PR?

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**